Index: usbmuxd/libusbmuxd.pc.in
===================================================================
--- usbmuxd.orig/libusbmuxd.pc.in	2010-01-15 15:35:03.000000000 +0100
+++ usbmuxd/libusbmuxd.pc.in	2010-01-15 15:35:29.000000000 +0100
@@ -6,7 +6,7 @@
 Name: libusbmuxd
 Description: A library to communicate with the usbmux daemon
 Version: ${USBMUXD_VERSION}
-Requires: libusb-1.0 >= 1.0.3
+Requires.private: libusb-1.0 >= 1.0.3
 Libs: -L${CMAKE_INSTALL_PREFIX}/lib${LIB_SUFFIX} -lusbmuxd
 Cflags: -I${CMAKE_INSTALL_PREFIX}/include
 
