From 00335c0197c3e130d7002ebab2a808a9433b3233 Mon Sep 17 00:00:00 2001
From: Martin S
Date: Mon, 14 Mar 2011 14:25:47 +0000
Subject: Bump PID range for iPad 2, iPhone 5 and their variants

---
Index: usbmuxd/daemon/usb.h
===================================================================
--- usbmuxd.orig/daemon/usb.h	2011-03-20 14:02:18.000000000 +0100
+++ usbmuxd/daemon/usb.h	2011-03-20 14:03:32.000000000 +0100
@@ -46,7 +46,7 @@
 
 #define VID_APPLE 0x5ac
 #define PID_RANGE_LOW 0x1290
-#define PID_RANGE_MAX 0x129f
+#define PID_RANGE_MAX 0x12af
 
 struct usb_device;
 
Index: usbmuxd/udev/85-usbmuxd.rules.in
===================================================================
--- usbmuxd.orig/udev/85-usbmuxd.rules.in	2011-03-20 14:04:00.000000000 +0100
+++ usbmuxd/udev/85-usbmuxd.rules.in	2011-03-20 14:05:21.000000000 +0100
@@ -1,7 +1,7 @@
 # usbmuxd (iPhone "Apple Mobile Device" MUXer listening on /var/run/usbmuxd)
 
-# Forces iPhone 1.0, 3G, 3GS, iPodTouch 1/2 and iPad to USB configuration 3 and run usbmuxd
-ACTION=="add", SUBSYSTEM=="usb", OWNER="usbmux", ATTR{idVendor}=="05ac", ATTR{idProduct}=="129[0-9a-f]", ENV{USBMUX_SUPPORTED}="1", ATTR{bConfigurationValue}!="$attr{bNumConfigurations}", ATTR{bConfigurationValue}="$attr{bNumConfigurations}", RUN+="@CMAKE_INSTALL_PREFIX@/sbin/usbmuxd -u -U usbmux"
+# Forces iDevices to the last USB configuration and runs usbmuxd
+ACTION=="add", SUBSYSTEM=="usb", OWNER="usbmux", ATTR{idVendor}=="05ac", ATTR{idProduct}=="12[9a][0-9a-f]", ENV{USBMUX_SUPPORTED}="1", ATTR{bConfigurationValue}!="$attr{bNumConfigurations}", ATTR{bConfigurationValue}="$attr{bNumConfigurations}", RUN+="@CMAKE_INSTALL_PREFIX@/sbin/usbmuxd -u -U usbmux"
 
 # Exit usbmuxd when the last device is removed
-ACTION=="remove", SUBSYSTEM=="usb", ENV{PRODUCT}=="5ac/129[0-9a-f]/*", ENV{INTERFACE}=="255/*", RUN+="@CMAKE_INSTALL_PREFIX@/sbin/usbmuxd -x"
+ACTION=="remove", SUBSYSTEM=="usb", ENV{PRODUCT}=="5ac/12[9a][0-9a-f]/*", ENV{INTERFACE}=="255/*", RUN+="@CMAKE_INSTALL_PREFIX@/sbin/usbmuxd -x"
